src/Components/button.js

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name, size, last, clickHandler } = props;
+  const {
+    name, size, last, clickHandler,
+  } = props;
 
-  function handleClick(_name)  {
+  function handleClick() {
     clickHandler(name);
   }
```

Comment:
this has to be a pure function, please get used to passing a variable you've using inside a function ( `name`) as a **parameter** to that function
```
handleClick = (buttonName) => {
  this.props.clickHandler(buttonName);
}
```

Reply:
Pls read this. i think eslint wont like if i do that. that was in first place the reason why i changed from that to this.

Reply:
D:\gitpublic\microverse\muffin-calculator\src\App.js
  13:15  error  Parsing error: Unexpected token =

See?

Reply:
5:17  error  Parse errors in imported module './Components/App': Unexpected token = (13:15)  import/no-named-as-default
  5:17  error  Parse errors in imported module './Components/App': Unexpected token = (13:15)  import/no-named-as-default-member

D:\gitpublic\microverse\muffin-calculator\src\Components\App.js
  13:15  error  Parsing error: Unexpected token =

D:\gitpublic\microverse\muffin-calculator\src\index.js
  6:17  error  Parse errors in imported module './Components/App': Unexpected token = (13:15)  import/no-named-as-default
  6:17  error  Parse errors in imported module './Components/App': Unexpected token = (13:15)  import/no-named-as-default-member


al them just because of it @shloch See?

Reply:
`4:1  error  Component should be written as a pure function  react/prefer-stateless-function`